src/parser/transform/statement/transform_select_node.cpp

```diff
@@ -101,6 +102,10 @@ unique_ptr<QueryNode> Transformer::TransformSelectNode(duckdb_libpgquery::PGSele
 		case duckdb_libpgquery::PG_SETOP_INTERSECT:
 			result->setop_type = SetOperationType::INTERSECT;
 			break;
+		case duckdb_libpgquery::PG_SETOP_UNION_BY_NAME:
+			select_distinct = false;
```

Comment:
Should we perhaps allow `UNION BY NAME` and `UNION ALL BY NAME`, to be more consistent w.r.t. the standard SQL syntax?

Reply:
finished